src/ngraph/builder/quantization_util.hpp

```diff
@@ -95,11 +96,33 @@ namespace ngraph
                 // s32 = f32 * std::pow(2, 31)/ max_abs32;
                 // s8 = f32 * std::pow(2, 7)/ max_abs8;
                 // s8 = s32 * std::pow(2, -24) * max_abs32 / max_abs8;
-                const float scale = static_cast<float>(
-                    (std::pow(2, -24) * static_cast<double>(max_abs32 / max_abs8)));
+                float scale = static_cast<float>((std::pow(2, is_output_signed ? -24 : -23) *
+                                                  static_cast<double>(max_abs32 / max_abs8)));
                 return scale;
             }
 
+            float get_bias_scale(const std::shared_ptr<Node> min_input,
+                                 const std::shared_ptr<Node> max_input,
+                                 const std::shared_ptr<Node> min_filter,
+                                 const std::shared_ptr<Node> max_filter)
+            {
+                auto min_input_const_op = std::static_pointer_cast<ngraph::op::Constant>(min_input);
+                auto max_input_const_op = std::static_pointer_cast<ngraph::op::Constant>(max_input);
+                auto min_filter_const_op =
+                    std::static_pointer_cast<ngraph::op::Constant>(min_filter);
+                auto max_filter_const_op =
+                    std::static_pointer_cast<ngraph::op::Constant>(max_filter);
+                auto input_min = min_input_const_op->get_vector<float>();
+                auto input_max = max_input_const_op->get_vector<float>();
+                auto filter_min = min_filter_const_op->get_vector<float>();
+                auto filter_max = max_filter_const_op->get_vector<float>();
+
+                float bias_scale =
+                    255.0 * 127.0 / (std::max(std::abs(input_min[0]), std::abs(input_max[0])) *
```

Comment:
255.0 and 127.0 are the std::numeric_limits<uint8_t>::max() and std::numeric_limits<uint8_t>::min(), right?  we should use that here instead of these magic numbers.  Note that soon I want to add `min` and `max` to element::Type so that these values can be defined in the nGraph type system.  not asking you to make that change.  just remove the magic numbers.